packages/ndla-ui/src/Resource/BlockResource.tsx

```diff
@@ -186,11 +183,15 @@ const BlockResource = ({
         <ContentWrapper>
           <ResourceTypeAndTitleLoader loading={isLoading}>
             <ResourceTitleLink data-link="" title={title} target={targetBlank ? '_blank' : undefined} to={link}>
-              <Heading css={resourceHeadingStyle}>{title}</Heading>
+              <Text element="label" textStyle="label-small" css={resourceHeadingStyle}>
```

Comment:
Dette er nok ikke en label. Label skal som oftest knyttes opp mot form-komponenter.

Reply:
Det var det Hedvig ville. Dette blir akk som Arena/Topic kortene, der titellen er en label

Reply:
Om jeg leser issuet riktig, så er det bare at vi ikke må bruke H1. H1 skal alltid være primæroverskrifta på sida. Men headers skal alltid være hx, og de skal følge i synkende rekkefølge (dvs aldri hoppe fra 1 til 3, f.eks.).
Labels er til bruk for å gi beskrivelse til et spesifikt element (vanligvis input, select, textfield), men ikke som overskrift.

Reply:
Label style kan vi gi Hedvig uten å gi label-element. HTML-koden må fortsatt være semantisk riktig for at en skjermleser skal oppfatte dem rett, og der tolkes labels og headers noe forskjellig :)

Reply:
Tanken her er at dette ikke skal være en heading, så element kan byttes til span eller noe annet. Må bare være en Text komponent for å gi Label styling